chap-isaref-riscv.tex

```diff
@@ -117,34 +122,40 @@ \subsection*{Types used in function definitions}
 \sailRISCVval{capBitsToCapability}
 \sailRISCVval{capToBits}
 
-\subsection*{Functions for reading and writing register and memory}
+\subsection*{Functions for reading and writing registers and memory}
 
-\arnote{TODO: document X() and C()}
 %\label{sailRISCVzC}
 \begin{lstlisting}[language=sail,label=sailRISCVzC]
 C(n) : regno -> Capability
 C(n) : (regno, Capability) -> unit
 \end{lstlisting}
+\hspace{\parindent}
 The overloaded function \isail{C(n)} is used to read or write capability register \isail{n}.
 
+\medskip
 \begin{lstlisting}[language=sail,label=sailRISCVzX]
 X(n) : regno -> xlenbits
 X(n) : (regno, xlenbits) -> unit
 \end{lstlisting}
+\hspace{\parindent}
 The overloaded function \isail{X(n)} is used to read or write integer register \isail{n}.
 
+\medskip
 \begin{lstlisting}[language=sail,label=sailRISCVzF]
 F(n) : regno -> xlenbits
 F(n) : (regno, xlenbits) -> unit
 \end{lstlisting}
+\hspace{\parindent}
 The overloaded function \isail{F(n)} is used to read or write floating-point register \isail{n}.
 
 \medskip
 \sailRISCVval{memBitsToCapability}
 \sailRISCVval{capToMemBits}
 \sailRISCVval{int\_to\_cap}
+\sailRISCVval{mem\_read\_cap}
 
 \medskip
+\sailRISCVval{ddc\_and\_resulting\_addr}
```

Comment:
This function could possibly use a description (maybe ideally in Sail that auto-populates here?)

Reply:
There is a doc comment, so sail should generated the docs for this?

Reply:
Ah I think this is the LaTex output issue where the doc comment needs to be on a `val` and the `func` comment is ignored.